README.md

```diff
@@ -276,7 +274,9 @@ The available releases are:
 ...
 26.0.2 *
 ```
-<!-- markdownlint-enable MD007 # line-length -->
+
+**Note**: this list, kept in a file managed by Kerl, is different depending on the build backend
```

Comment:
`kerl` no uppercasing please

Reply:
Sure. It was there already, so I took "advantage" of that. Maybe I'll take this chance to lowercase it all, then (?)

Reply:
f7fcb26